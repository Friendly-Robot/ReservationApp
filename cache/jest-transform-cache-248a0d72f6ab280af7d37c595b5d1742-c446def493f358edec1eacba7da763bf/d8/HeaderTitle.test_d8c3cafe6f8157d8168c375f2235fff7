f9bee0108d7294959defb10163cd6943
var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

require("react-native");

var _react = _interopRequireDefault(require("react"));

var _HeaderTitle = _interopRequireDefault(require("../HeaderTitle"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

it('renders correctly', function () {
  var tree = _reactTestRenderer.default.create(_react.default.createElement(_HeaderTitle.default, null)).toJSON();

  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkhlYWRlclRpdGxlLnRlc3QuanMiXSwibmFtZXMiOlsiaXQiLCJ0cmVlIiwicmVuZGVyZXIiLCJjcmVhdGUiLCJ0b0pTT04iLCJleHBlY3QiLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiI7O0FBSUE7O0FBQ0E7O0FBQ0E7O0FBR0E7O0FBRUFBLEVBQUUsQ0FBQyxtQkFBRCxFQUFzQixZQUFNO0FBQzVCLE1BQU1DLElBQUksR0FBR0MsMkJBQVNDLE1BQVQsQ0FBZ0IsNkJBQUMsb0JBQUQsT0FBaEIsRUFBaUNDLE1BQWpDLEVBQWI7O0FBQ0FDLEVBQUFBLE1BQU0sQ0FBQ0osSUFBRCxDQUFOLENBQWFLLGVBQWI7QUFDRCxDQUhDLENBQUYiLCJzb3VyY2VzQ29udGVudCI6WyIvKipcbiAqIEBmb3JtYXRcbiAqL1xuXG5pbXBvcnQgJ3JlYWN0LW5hdGl2ZSc7XG5pbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnO1xuaW1wb3J0IEhlYWRlclRpdGxlIGZyb20gJy4uL0hlYWRlclRpdGxlJztcblxuLy8gTm90ZTogdGVzdCByZW5kZXJlciBtdXN0IGJlIHJlcXVpcmVkIGFmdGVyIHJlYWN0LW5hdGl2ZS5cbmltcG9ydCByZW5kZXJlciBmcm9tICdyZWFjdC10ZXN0LXJlbmRlcmVyJztcblxuaXQoJ3JlbmRlcnMgY29ycmVjdGx5JywgKCkgPT4ge1xuICBjb25zdCB0cmVlID0gcmVuZGVyZXIuY3JlYXRlKDxIZWFkZXJUaXRsZSAvPikudG9KU09OKCk7XG4gIGV4cGVjdCh0cmVlKS50b01hdGNoU25hcHNob3QoKTtcbn0pO1xuIl19