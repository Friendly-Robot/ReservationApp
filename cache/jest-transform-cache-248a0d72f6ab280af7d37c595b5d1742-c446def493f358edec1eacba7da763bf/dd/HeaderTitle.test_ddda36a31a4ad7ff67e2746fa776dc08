63f9902542b643d7362e8086975171a2
var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

require("react-native");

var _react = _interopRequireDefault(require("react"));

var _HeaderTitle = _interopRequireDefault(require("../HeaderTitle"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

describe('<HeaderTitle />', function () {
  it('should be defined', function () {
    expect(_HeaderTitle.default).toBeDefined();
  });
  it('renders correctly', function () {
    var tree = _reactTestRenderer.default.create(_react.default.createElement(_HeaderTitle.default, null)).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkhlYWRlclRpdGxlLnRlc3QuanMiXSwibmFtZXMiOlsiZGVzY3JpYmUiLCJpdCIsImV4cGVjdCIsIkhlYWRlclRpdGxlIiwidG9CZURlZmluZWQiLCJ0cmVlIiwicmVuZGVyZXIiLCJjcmVhdGUiLCJ0b0pTT04iLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiI7O0FBSUE7O0FBQ0E7O0FBQ0E7O0FBR0E7O0FBRUFBLFFBQVEsQ0FBQyxpQkFBRCxFQUFvQixZQUFNO0FBQ2hDQyxFQUFBQSxFQUFFLENBQUMsbUJBQUQsRUFBc0IsWUFBTTtBQUM1QkMsSUFBQUEsTUFBTSxDQUFDQyxvQkFBRCxDQUFOLENBQW9CQyxXQUFwQjtBQUNELEdBRkMsQ0FBRjtBQUlBSCxFQUFBQSxFQUFFLENBQUMsbUJBQUQsRUFBc0IsWUFBTTtBQUM1QixRQUFNSSxJQUFJLEdBQUdDLDJCQUFTQyxNQUFULENBQWdCLDZCQUFDLG9CQUFELE9BQWhCLEVBQWlDQyxNQUFqQyxFQUFiOztBQUNBTixJQUFBQSxNQUFNLENBQUNHLElBQUQsQ0FBTixDQUFhSSxlQUFiO0FBQ0QsR0FIQyxDQUFGO0FBSUQsQ0FUTyxDQUFSIiwic291cmNlc0NvbnRlbnQiOlsiLyoqXG4gKiBAZm9ybWF0XG4gKi9cblxuaW1wb3J0ICdyZWFjdC1uYXRpdmUnO1xuaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0JztcbmltcG9ydCBIZWFkZXJUaXRsZSBmcm9tICcuLi9IZWFkZXJUaXRsZSc7XG5cbi8vIE5vdGU6IHRlc3QgcmVuZGVyZXIgbXVzdCBiZSByZXF1aXJlZCBhZnRlciByZWFjdC1uYXRpdmUuXG5pbXBvcnQgcmVuZGVyZXIgZnJvbSAncmVhY3QtdGVzdC1yZW5kZXJlcic7XG5cbmRlc2NyaWJlKCc8SGVhZGVyVGl0bGUgLz4nLCAoKSA9PiB7XG4gIGl0KCdzaG91bGQgYmUgZGVmaW5lZCcsICgpID0+IHtcbiAgICBleHBlY3QoSGVhZGVyVGl0bGUpLnRvQmVEZWZpbmVkKCk7XG4gIH0pO1xuXG4gIGl0KCdyZW5kZXJzIGNvcnJlY3RseScsICgpID0+IHtcbiAgICBjb25zdCB0cmVlID0gcmVuZGVyZXIuY3JlYXRlKDxIZWFkZXJUaXRsZSAvPikudG9KU09OKCk7XG4gICAgZXhwZWN0KHRyZWUpLnRvTWF0Y2hTbmFwc2hvdCgpO1xuICB9KTtcbn0pXG4iXX0=