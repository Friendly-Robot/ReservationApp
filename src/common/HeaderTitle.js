/*
* @flow
*/

import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

import theme from '../theme';

type titleProps = {
  title: string,
};

export default HeaderTitle = ({ title }) => (
  <View style={styles.container}>
    <Text style={styles.title}>{ title }</Text>
  </View>
);

const styles = StyleSheet.create({
  container: {
    backgroundColor: theme.COLOR_WHITE,
    height: 65,
    justifyContent: 'center',
    zIndex: 10,
  },
  title: {
    color: theme.COLOR_PRIMARY,
    fontSize: theme.FONT_SIZE_LARGE,
    marginLeft: theme.BORDER_PADDING_HORIZONTAL,
  },
});