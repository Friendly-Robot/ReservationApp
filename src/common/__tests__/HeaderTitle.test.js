/**
 * @format
 */

import 'react-native';
import React from 'react';
import HeaderTitle from '../HeaderTitle';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<HeaderTitle />).toJSON();
  expect(tree).toMatchSnapshot();
});
