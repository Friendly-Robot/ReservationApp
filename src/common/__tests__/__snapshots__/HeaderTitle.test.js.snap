// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgb(255, 255, 255)",
      "flexDirection": "row",
      "height": 65,
      "zIndex": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "rgb(30, 147, 242)",
        "fontSize": 16,
        "marginLeft": 15,
      }
    }
  >
    
  </Text>
</View>
`;
